.github/CODEOWNERS

```diff
@@ -1,5 +1,26 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# What is a "CODEOWNER"?
+#
+# A CODEOWNER MUST ____ for packages they own:
+# - introduce themselves at a Python SIG meeting
```

Comment:
```suggestion
```
I think this has a good intention because it can help collaboration, but the SIG meeting may be hard to attend for many people because of scheduling conflicts.

Reply:
Hm but I think it's optimistic that we will have people outside of the SIG wanting to be "CODEOWNERS" anyways 😝 . Joining 1 meeting shouldn't be too hard of an ask, and I think it's important to at least hear "why" they want to take a package to 1.0 so that the maintainers can feel a little at ease that the person has the understanding to do so and they won't just take off after the package gets released 🙃

Reply:
Our time slot is pretty difficult for folks in APAC

Reply:
Yes, that is important as well. I'll be happy with contributors just introducing themselves in the Slack channel.

Reply:
I think a Slack introduction is a great solution, I'll update it to be so.